x/incentives/keeper/distribute_test.go

```diff
@@ -177,80 +185,107 @@ func (suite *KeeperTestSuite) TestSyntheticDistribute() {
 
 }
 
-// TestGetModuleToDistributeCoins tests the sum of coins yet to be distributed for all of the module is correct.
-func (suite *KeeperTestSuite) TestGetModuleToDistributeCoins() {
-	suite.SetupTest()
+// TestGetModuleToDistributeAndDistributedCoins tests the sum of coins yet to be distributed
+// and the distributed coins is correct.
+func (suite *KeeperTestSuite) TestGetModuleToDistributeAndDistributedCoins() {
```

Comment:
Why are we merging tests for `GetModuleDistributedCoins(...)` and `GetModuleToDistributeCoins(...)`?

Although the setup logic / testcases might be similar, they should still have separate fixtures. A test needs to test one unit of computation and do it well.

Also, I'm not seeing where `GetModuleDistributedCoins(...)` is now tested? I think it might have been removed and untested in general.

Reply:
ohhh, that is indeed missing, I'll add it

Reply:
I've separated the 2 tests